LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2009-2016 Homebrew contributors.
+Copyright 2009-2017 Homebrew contributors.
```

Comment:
@maxim-belkin, interesting question. I've just been using "Homebrew" everywhere, even on code that originated entirely with us and not with upstream, to make it easier to copy code around between repos.

I don't personally believe it's necessary to add a Linuxbrew-specific line, but I'm not opposed to it either.